components/grid/col.tsx

```diff
@@ -17,6 +18,7 @@ export interface ColSize {
 }
 
 export interface ColProps extends React.HTMLAttributes<HTMLDivElement> {
+  flex?: FlexType;
```

Comment:
I moved this one up to follow the same order as [the docs](https://ant.design/components/grid/#Col)